### DISCUSSION TITLE
Fatal error

### DISCUSSION THREAD

**Discussion by KeilaAlmeyra** [INGCOM-UNRN-P1/cursada-2023#57] [Preguntas y respuestas] [answered]
Buenas, ¿cuál podría ser la causa y solución a un fatal error de este estilo? 
```
: fatal error: stdio.h: No such file or directory
    8 | #include <stdio.h>
      |          ^~~~~~~~~
compilation terminated.
```
Me parece raro porque con el anterior, que lo hice de la misma forma (abriendo una carpeta nueva e iniciando ahí Git Bush) no me pasó. Intenté arreglarlo con moverlo al disco c y también con abrir y cerrar pero nada.

**Comment by martinvilu** [comment]
¿Qué tenés instalado? ¿qué guía has seguido para instalar y configurar las herramientas?

**Reply by KeilaAlmeyra** [reply]
Tengo instalado Visual Studio Code (no lo uso), el compilador, editor de texto Notepad y Git (donde estuve haciendo los ejercicios, Git Bash). La guía que usé fue la de Control de Versiones con Git y GitHub, menos para la parte de la llave que la hice en base a un tutorial.

**Reply by martinvilu** [reply]
Como configuraste el compilador? (lo referido al `path`), mandá tambien una captura de la salida de `env` ejecutado en una terminal

**Comment by KeilaAlmeyra** [comment]
Lo único que hice sobre el path, fue lo que decía en la guía en la parte de ir a variables de entorno y agregar aquello en él. Como luego daba bien la versión y pude hacer lo de la ficha, supuse que ya estaba bien

![Captura de pantalla (92)](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141676780/185e9656-106e-4f8d-89ab-8498812ab21e)
![Captura de pantalla (93)](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141676780/5773756e-7778-426e-a4c9-c42dab2aa616)

**Reply by martinvilu** [reply]
Subí el código al repositorio, en particular el que esta fallando y compartí aca la dirección del repositorio cuando este.

**Reply by KeilaAlmeyra** [reply]
https://github.com/INGCOM-UNRN-P1/tp1-2023-KeilaAlmeyra

Listo, profe

**Reply by dteira** [reply]
Hola, 
¿sigue abierto el problema?. Me parece que es algo del ambiente porque cloné el repo y pude compilar sin problemas. 
Si sigue, en la consola git bash;
- ejecutá "env | grep PATH" y subí lo que muestra. 
- ejecutá "gcc" y subí lo que muestra.

Pudiste compilar el ejercicio 1?

Saludos,
Daniel

**Reply by KeilaAlmeyra** [reply]
Hola, buenos días

Con el ejercicio 1 no tuve problemas, fue a partir del 2 que me apareció y no se salió más. Lo siguiente que intenté por recomendación de compañeros fue intentar seguir en Visual y desinstalar e instalar otra vez pero persiste error (hasta en el Visual). 

Le adjunto lo pedido:
![Captura de pantalla (94)](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141676780/1889af26-7877-47e1-9328-a764fd518ef0)

**Reply by dteira** [reply]
Bien se ve el patch al directorio INCLUDE. Y si intentás compilar con "gcc ejercicio2.c -o ejercicio2.exe" en ese mismo lugar que sale por pantalla?. 
mandá captura si podés.
Saludos,
Daniel.

**Reply by KeilaAlmeyra** [reply]
Ahí lo intenté en el que me indicó y en otro directorio que encontré que también tenía un archivo con el nombre de "include", ¿está bien así como lo hice o era en otra parte que debía hacerlo?
Saludos,
Keila. 

![Captura de pantalla (98)](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141676780/59782acc-f76f-46ee-8244-aeeaba1fa376)
![Captura de pantalla (96)](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141676780/a8c630c6-4bfd-436d-8319-fcb7cabf693b)

**Comment by martinvilu** [comment, marked as answer]
Anduvo? Lo podemos dar por cerrado? Cómo hiciste para solucionarlo?

**Reply by martinvilu** [reply]
Si está cerrado, dale click a
![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/b0a477ac-0c7a-462b-a36b-b9b09fa3c698)

Sino, seguimos conversando del tema
